ani-cli

```diff
@@ -382,15 +382,16 @@ open_episode () {
 	# cool way of clearing screen
 	tput clear
 	# checking if episode is in range
-	while [ "$episode" -gt "$last_ep_number" ] || [ -z "$episode" ]; do
+	while [ "$episode" -gt "$last_ep_number" ] || [ -z "$episode" ] || [ "$episode" -lt "$first_ep_number" ]; do
 	is_number "$ep_choice_start"
 		if [ "$last_ep_number" -eq 0 ]; then
 			die "Episodes not released yet!"
 		else
 			err "Episode out of range"
 		fi
 		prompt "Choose episode" "[$first_ep_number-$last_ep_number]"
-		episode="$REPLY $REPLY2"
+		episode="$REPLY"
+		ep_choice_start=$episode
```

Comment:
This probably breaks range selection

Reply:
If I understand correctly range selection is not possible with episode variable anyways correct me if I'm wrong

Reply:
this breaks the range selection